modules/microshift-gitops-adding-apps.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-adding-apps_{context}"]
+= Creating GitOps applications on {microshift-short}
+
+You can create a custom YAML configuration to deploy and manage applications in your {microshift-short} service.
+
+.Prerequisites 
+
+* You installed the GitOps ArgoCD package
+
+.Procedure 
+
+. Create a YAML file and add your customized configurations for the application. 
++
+[source,terminal]
+----
+$ oc create -f <filename>.yaml
+----
+
```

Comment:
```suggestion
```
Remove this lines because in the step 1 consist only on creating the yaml file, step 2 will run the `oc` command